### ISSUE TITLE
problem about pretrain model's inputs

### ISSUE THREAD

**Issue by zyxunh** [CSAILVision/semantic-segmentation-pytorch#81]
The default setting input of pytorch's pretrain model is  in the range [-1.0, 1.0]. But I see this code just subtract the mean without scale normalization,  is my understanding correct?

**Comment by Tete-Xiao** [comment]
We use our own pre-trained models. Please refer to README for more details.
